### PR TITLE
Update syslog-notify-oxidized.php

### PR DESCRIPTION
**Pull request by kruczek8989**
For oxidized to be able to catch from syslog a configuration change on a CISCO switch via SNMP.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: scripts/syslog-notify-oxidized.php [hunk 1]
```diff
@@ -11,6 +11,8 @@
 $oxidized_api = new \App\ApiClients\Oxidized();
 if (preg_match('/(SYS-(SW[0-9]+-)?5-CONFIG_I|VSHD-5-VSHD_SYSLOG_CONFIG_I): Configured from .+ by (?P<user>.+) on .*/', $msg, $matches)) {
     $oxidized_api->updateNode($hostname, $msg, $matches['user']);
+} elseif (preg_match('/(SYS-(SW[0-9]+-)?5-CONFIG_I|VSHD-5-VSHD_SYSLOG_CONFIG_I): Configured from .+ by.*/', $msg, $matches)) {
```
[the remark this answers is not in the document]
Comment (kruczek8989) [reply scripts/syslog-notify-oxidized.php:14]: The difference is the end of the command that appears in syslog.

While a configuration change is made via SNMP in syslog, 
I get this message:
Program: "% SYS-5-CONFIG_I"
Message: "Configured from <IP_address> by snmp"

You could possibly remodel this first entry and shorten it by "(? P <user>. +) on".

[the remark this answers is not in the document]
Comment (murrant) [reply scripts/syslog-notify-oxidized.php:14]: Here is what I would change the first command to:
`/(SYS-(SW[0-9]+-)?5-CONFIG_I|VSHD-5-VSHD_SYSLOG_CONFIG_I): Configured from .+ by (?P<user>.+)( on .*)?$/`

[the remark this answers is not in the document]
Comment (murrant) [reply scripts/syslog-notify-oxidized.php:14]: This means it either must have " on " after the user, or be the end of the line.

[the remark this answers is not in the document]
Comment (murrant) [reply scripts/syslog-notify-oxidized.php:14]: Actually, I forgot to change the user capture to non-space characters (\S).  Edited above

[the remark this answers is not in the document]
Comment (kruczek8989) [reply scripts/syslog-notify-oxidized.php:14]: That's ok and working ;)
Your solution is better ;)

**Review by kruczek8989 — COMMENTED** [review]
Ok

**Review by murrant — APPROVED** [review]
Thanks!

**Pull request merged**